src/main/java/domain/Car.java

```diff
@@ -0,0 +1,66 @@
+package domain;
+
+import java.util.Comparator;
+import java.util.Objects;
+
+public class Car {
+    public static final int LEAST_CONDITION = 4;
+    public static final int MAX_NAME_LENGTH = 5;
+    public static final int START_POSITION = 0;
+    public static final int ONE_STEP = 1;
+    private final String name;
+    private int position;
+    public static final Comparator<Car> positionComparator = new Comparator<Car>() {
+        @Override
+        public int compare(Car o1, Car o2) {
+            return o1.position - o2.position;
+        }
+    };
+
+    public Car(String name) {
+        validateName(name);
```

Comment:
```suggestion
        validate(name);
```

메서드에 넘겨주는 값만 봐도 어떤 것을 validate하는지 충분히 알 수 있지 않을까요?